modules/backend/models/UserThrottle.php

```diff
@@ -21,4 +22,12 @@ class UserThrottle extends ThrottleBase
     public $belongsTo = [
         'user' => User::class
     ];
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        static::$attemptLimit = Config::get('auth.attemptLimit', 5);
+        static::$suspensionTime = Config::get('auth.suspensionTime', 15);
+    }
```

Comment:
That won't work in static context...

Reply:
Ok, so since this is used in non-static context, it'll work.
ref. https://github.com/octobercms/library/blob/master/src/Auth/Models/Throttle.php#L77-L88
ref. https://github.com/octobercms/library/blob/master/src/Auth/Models/Throttle.php#L213-L227
ref. https://github.com/octobercms/library/blob/master/src/Auth/Models/Throttle.php#L236-L249